docs/content/kv_commands.md

```diff
@@ -39,12 +64,13 @@ yes
 
 ### `rename`
 Renames a given namespace.
+Takes an optional `--env` argument.
 
 #### Usage
 
 ```sh
-$ wrangler kv:namespace rename f7b02e7fc70443149ac906dd81ec1791 "updated kv namespace"
-🌀  Renaming namespace f7b02e7fc70443149ac906dd81ec1791 with title "updated kv namespace"
+$ wrangler kv:namespace rename OLD_NAMESPACE NEW_NAMESPACE
```

Comment:
nit: weirdness here is that `OLD_NAMESPACE` is a binding, but `NEW_NAMESPACE` is a title. perhaps change the convention for the latter to differentiate, like `"New Namespace"`
```suggestion
$ wrangler kv:namespace rename OLD_NAMESPACE "New Namespace"
```

Reply:
titles are gonna be the death of mt